.github/workflows/install.yml

```diff
@@ -0,0 +1,18 @@
+name: Run installer tests
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
In a separate PR It'd be good to test this on macOS too, but I think that's a bit less important since the team is regularly testing macOS. For Windows with WSL2 we probably want to make sure the `/usr/local/{bin,lib}` detection works too.

Reply:
I've added macOS to this test now. I can do WSL in a separate PR once I figure out how to invoke it from a GH action :)